test/integration_test.go

```diff
@@ -103,6 +111,12 @@ func TestIntegration(t *testing.T) {
 			prettyPrintWhoCanOutput(t, tt.args, out)
 
 			for _, line := range tt.output {
+				// TODO Improve asserts on the output
```

Comment:
We need a separate PR for that. In this PR I just wanted to experiment with KinD GitHub Action.